### PR TITLE
remove unnecessary columns from account management table; redirect premium_hub url to new page; clean up dead code

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Final pieces of the Premium Hub Overview/Account Management project, including: removing two unnecessary columns from the table on Account Management, and its query as well; redirecting the `/premium_hub` url to the new Overview component; and removing all the old code for that page now that it's no longer used.

## WHY
Removing these two columns gives us some speed improvements, and the data is redundant with the Usage Snapshot. This completes this project, so we're ready to make it public-facing.

### Screenshots
Screenshot 2024-01-09 at 8 33 29 AM

### Notion Card Links
https://www.notion.so/quill/On-the-Premium-Hub-Account-Management-page-remove-the-Activities-Completed-and-Time-Spent-co-7149a8c4d6fb4b4dbee374c31fdee16e?pvs=4

https://www.notion.so/quill/Clean-up-old-premium-hub-overview-page-code-49a3fde96fce4e0d969024fe97b0e9ce?pvs=4

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | YES
Have you deployed to Staging? | YES
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | YES

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/PremiumHub/components/overview/premiumReportsSection.tsx [hunk 1]
```diff
@@ -12,7 +14,7 @@ export const premiumReportTiles = [
     new: true
   },
   {
-    name: 'Diagnostic Growth Report',
+    name: DIAGNOSTIC_GROWTH_REPORT,
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/PremiumHub/components/overview/premiumReportsSection.tsx:17]: Yay making things constants.

**Review by anathomical — APPROVED** [review]

File: services/QuillLMS/client/app/bundles/PremiumHub/components/overview/premiumReportsSection.tsx [hunk 2]
```diff
@@ -47,14 +49,16 @@ export const premiumReportTiles = [
...
+    const button = tile.name === DIAGNOSTIC_GROWTH_REPORT ? <button className={`${buttonClassName} disabled`} disabled={true} type="button">Coming soon</button> : <Link className={buttonClassName} to={tile.link}>View report</Link>
     return (
       <div className={`tile ${tile.new ? 'new' : ''}`} key={tile.name}>
         <div>
           <h3>{tile.name}{tile.new ? <span className="new-tag">NEW</span> : null}</h3>
           <p>{tile.description}</p>
         </div>
         <div className="link-and-image">
-          <Link className="quill-button focus-on-light outlined secondary medium" to={tile.link}>View report</Link>
+          {button}
```
Comment (happythenewsad) [inline services/QuillLMS/client/app/bundles/PremiumHub/components/overview/premiumReportsSection.tsx:61]: 👍

**Review by happythenewsad — APPROVED** [review]
🧹 🧹 🧹

**Review by eadams17 — APPROVED** [review]
love all of this cleanup!

**Pull request merged**